tests/toranj/build.sh

```diff
@@ -219,7 +220,7 @@ case ${build_config} in
         cd "${top_builddir}" || die "cd failed"
         cmake -GNinja -DOT_PLATFORM=simulation -DOT_COMPILE_WARNING_AS_ERROR=ON -DOT_COVERAGE=${ot_coverage} \
             -DOT_THREAD_VERSION=1.4 -DOT_APP_CLI=ON -DOT_APP_NCP=OFF -DOT_APP_RCP=OFF \
-            -DOT_15_4=ON -DOT_TREL=ON \
+            -DOT_15_4=ON -DOT_TREL=ON -DOT_CLI_IP6_RECV_CALLBACK=ON \
```

Comment:
We can remove these option if we add the #ifndef in `tests/toranj/openthread-core-toranj-config.h`. We can only add one in your another PR where we want to disable this option.